Build failure due to phantomjs dependency

Failed project:  Falcon UI

Error: 
[INFO] PhantomJS not found on PATH
[ERROR] Unexpected platform or architecture: linux/ppc64
[ERROR] It seems there is no binary available for your platform/architecture
[ERROR] Try to install PhantomJS globally
[ERROR] npm ERR! phantomjs@1.9.20 install: `node install.js`
[ERROR] npm ERR! Exit status 1
[ERROR] npm ERR!
[ERROR] npm ERR! Failed at the phantomjs@1.9.20 install script.
[ERROR] npm ERR! This is most likely a problem with the phantomjs package,
[ERROR] npm ERR! not with npm itself.
[ERROR] npm ERR! Tell the author that this fails on your system:
[ERROR] npm ERR!     node install.js
[ERROR] npm ERR! You can get their info via:
[ERROR] npm ERR!     npm owner ls phantomjs
[ERROR] npm ERR! There is likely additional logging output above.
[ERROR]
[ERROR] npm ERR! System Linux 3.16.0-30-generic
[ERROR] npm ERR! command "/ws/HDP-falcon/falcon-ui/node/node" "/ws/HDP-falcon/falcon-ui/node/node_modules/npm/bin/npm-cli.js" "install"
[ERROR] npm ERR! cwd /ws/HDP-falcon/falcon-ui
[ERROR] npm ERR! node -v v4.4.7
[ERROR] npm ERR! npm -v 1.4.3
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] npm ERR!
[ERROR] npm ERR! Additional logging details can be found in:
[ERROR] npm ERR!     /ws/HDP-falcon/falcon-ui/npm-debug.log
[ERROR] npm ERR! not ok code 0

--- a/falcon-ui/package.json
+++ b/falcon-ui/package.json
@@ -33,7 +33,6 @@
     "jasmine": "^2.0.1",
     "karma": "^0.12.23",
     "karma-jasmine": "^0.1.5",
-    "karma-phantomjs-launcher": "^0.1.4",
     "protractor": "^1.3.1"
   }
 }

